demo/grid-columns-demos.html

```diff
@@ -148,6 +145,22 @@ <h3>Hiding Columns</h3>
               root.firstElementChild.alt = rowData.item.name.first + ' ' + rowData.item.name.last;
               root.firstElementChild.src = rowData.item.picture.thumbnail;
             };
+            
+            const contextMenu = document.querySelector('vaadin-context-menu');
+            contextMenu.listenOn = document.querySelector('vaadin-button');
+            contextMenu.openOn = 'click';
+            contextMenu.renderer = function(root) {
+              root.innerHTML  = '<div style="display: flex; flex-direction: column;"></div>';
+              columns.forEach((column, i) => {
+                const checkbox = window.document.createElement('vaadin-checkbox');
+                checkbox.textContent = column.getAttribute('name');
+                checkbox.checked = !column.hidden;
+                checkbox.onchange = () => column.hidden = !checkbox.checked;
```

Comment:
let's use `addEventListener` API to listen to `change` and `click` events.

Reply:
Done